### ISSUE TITLE
Signal strength low

### ISSUE THREAD

**Issue by networkandcode** [tomaspinho/rtl8821ce#72]
Hi, There is a signal issue, with this driver on Ubuntu. It works when the WiFi router is very close. This is a new Lenovo laptop ideapad 330, and I am not sure if there is a hardware issue, or something with this driver or settings. I followed the instructions on README for installing the driver for RTL8821CE

**Comment by robertdaleweir** [comment]
I used the following to install Wifi on my new Lenovo L340 laptop and was successful in getting it working.
https://forums.fedoraforum.org/showthread.php?321616-Fedora-30-with-new-kernel-5-1-5-300-can-t-find-RTL8821CE-Wi-Fi-adapter

  I am using it on 5GHz and I am quite satisfied with it.  I am working within 20 feet of my router and it has a 200 Mb/s throughput according to the menu of its connection stats.  My exact install process on Fedora 30 is set out below:
` 
			-------->

Secure boot = Disabled

sudo dnf -y upgrade
sudo dnf install -y make git kernel-headers kernel-devel elfutils-libelf-devel
sudo dnf install -y dkms

reboot

su
git clone https://github.com/tomaspinho/rtl8821ce
cd rtl8821ce
chmod +x dkms-install.sh
chmod +x dkms-remove.sh
sudo ./dkms-install.sh 

	<------------
 `


  I am not sure about its performance except to say that I am very pleased with the fact that I was able to install it and that it works.  Hopefully this might be of some help to someone.
